packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -744,6 +745,13 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// {@macro flutter.material.Material.clipBehavior}
   final Clip? clipBehavior;
 
+  /// {@template flutter.material.appbar.actionsPadding}
+  /// The padding between the [actions] and the end of the AppBar.
+  ///
+  /// Defaults to zero.
```

Comment:
```suggestion
  /// Defaults to [EdgeInsets.zero].
```

Reply:
Either way is fine:) I usually just use `EdgeInsets.zero`.